Libs/RaptureAddins/ElasticSearch/src/test/java/rapture/elasticsearch/ElasticSearchSearchRepositoryTest.java

```diff
@@ -150,117 +142,133 @@ public void testSearchAndConvert() {
+        assertEquals(SearchRepoType.uri.toString(), r.getSearchHits().get(0).getIndexType());
+        assertEquals("series://" + seriesDocPath, r.getSearchHits().get(0).getId());
         assertEquals("series://" + seriesDocPath, r.getSearchHits().get(0).getUri());
-        assertEquals("{\"parts\":[\"x\",\"y\"],\"repo\":\"" + repo + "\",\"scheme\":\"series\"}", r.getSearchHits().get(0).getSource());
-        r = e.searchForRepoUris(Scheme.DOCUMENT.toString(), repo, null);
+        assertEquals("{\"parts\":[\"x\",\"y\"],\"repo\":\"" + authority + "\",\"scheme\":\"series\"}", r.getSearchHits().get(0).getSource());
+        r = e.searchForRepoUris(Scheme.DOCUMENT.toString(), authority, null);
         assertEquals(100, r.getTotal().longValue());
         assertEquals(10, r.getSearchHits().size());
-        assertEquals(SearchRepoType.URI.toString(), r.getSearchHits().get(0).getIndexType());
-        assertEquals(docDocPath, r.getSearchHits().get(0).getId());
+        assertEquals(SearchRepoType.uri.toString(), r.getSearchHits().get(0).getIndexType());
+        assertEquals("document://" + docDocPath, r.getSearchHits().get(0).getId());
         assertEquals("document://" + docDocPath, r.getSearchHits().get(0).getUri());
-        assertEquals("{\"parts\":[\"doc0\"],\"repo\":\"" + repo + "\",\"scheme\":\"document\"}", r.getSearchHits().get(0).getSource());
+        assertEquals("{\"parts\":[\"doc0\"],\"repo\":\"" + authority + "\",\"scheme\":\"document\"}", r.getSearchHits().get(0).getSource());
     }
 
     @Test
     public void testRemove() {
         insertTestDocs();
-        rapture.common.SearchResponse r = e.search(Arrays.asList(SearchRepoType.DOC.toString()), "trying out Elasticsearch");
+        rapture.common.SearchResponse r = e.search(Arrays.asList(Scheme.DOCUMENT.toString()), "trying out Elasticsearch");
         assertEquals(100, r.getTotal().longValue());
 
         e.remove(new RaptureURI("document://unittest/doc24"));
         e.remove(new RaptureURI("document://unittest/doc69"));
         e.remove(new RaptureURI("document://unittest/doc77"));
         e.refresh();
 
-        r = e.search(Arrays.asList(SearchRepoType.DOC.toString()), "trying out Elasticsearch");
+        r = e.search(Arrays.asList(Scheme.DOCUMENT.toString()), "trying out Elasticsearch");
         assertEquals(97, r.getTotal().longValue());
 
         SeriesUpdateObject s = new SeriesUpdateObject("removerepo/a/b/c", Arrays.asList("k1", "k2", "k3"), Arrays.asList(1.0, 2.0, 4.0));
         e.put(s);
         e.refresh();
 
-        r = e.search(Arrays.asList(SearchRepoType.SERIES.toString()), "4.0");
+        r = e.search(Arrays.asList(Scheme.SERIES.toString()), "4.0");
         assertEquals(1, r.getTotal().longValue());
 
         e.remove(new RaptureURI("series://removerepo/a/b/c"));
         e.refresh();
 
-        r = e.search(Arrays.asList(SearchRepoType.SERIES.toString()), "v3");
+        r = e.search(Arrays.asList(Scheme.SERIES.toString()), "v3");
```

Comment:
I understand that. You confirm that 4.0 is there. Then you delete the series, but you don't test that 4.0 has gone; you only test for the absence of something that was never there.